build.assets/Dockerfile-multiarch-devtoolset-gcc

```diff
@@ -0,0 +1,98 @@
+FROM centos:7 AS base
+
+# Automatically supplied by the Docker buildkit
+ARG TARGETARCH
+
+# Minimal build tool versions
+ARG GCC_VERSION=7.4.0
+ARG CMAKE3_VERSION=3.12.3
+ARG LLVM_VERSION=10.0.1
+
+# #############################################################################
+# Platform-specific customisation.
+# #############################################################################
+
+## ARM 32 #####################################################################
+FROM base AS platform-setup-arm
+ARG GCC_BUILD_FLAGS="--build arm-unknown-linux-gnueabihf --with-float=hard --with-fpu=vfpv3-d16 --with-mode=thumb"
+
+ARG YUM_UPDATE_FLAGS=""
+RUN echo "armhfp" > /etc/yum/vars/basearch && \
+    echo "armv7hl" > /etc/yum/vars/arch && \
+    echo "armv7hl-redhat-linux-gnu" > /etc/rpm/platform
+
+## ARM 64 #####################################################################
+FROM base AS platform-setup-arm64
+ARG GCC_BUILD_FLAGS="--build aarch64-unknown-linux-gnu"
+
+# Installing the kerenel packages causes the update to hang on aarch64, so we
+# skip upgrading them.
+ARG YUM_UPDATE_FLAGS="--exclude kernel-*"
+RUN echo "aarch64-redhat-linux-gnu" > /etc/rpm/platform
+
+## 386 ########################################################################
+FROM base AS platform-setup-386
+ARG GCC_BUILD_FLAGS="--build i386-unknown-linux-gnu"
+
+## AMD 64 #####################################################################
+FROM base AS platform-setup-amd64
+ARG GCC_BUILD_FLAGS="--build x86_64-unknown-linux-gnu"
+
+## GCC built from sources #####################################################
+FROM platform-setup-$TARGETARCH
+
+# Prerequestiques
+RUN yum update -y $YUM_UPDATE_FLAGS && \
+    yum install -y \
+        automake \
+        libtool \
+        flex \        
+        autoconf-archive \
+        git \
```

Comment:
The git from the centos repo doesn't work in GHA (v2.x is needed to support `git submodules`)

Reply:
We do not need the newer git inside the container anymore. Cloning is done by the GHA themselves on host OS.